tests/baselines/reference/jsDocDontBreakWithNamespaces.baseline

```diff
@@ -25,7 +25,7 @@
 // zee('');
 //       ^
 // | ----------------------------------------------------------------------
-// | zee(**arg0: any**, arg1: any, arg2: any): any
+// | zee(**arg0: any**, arg1: any): any
```

Comment:
This function is:

```
/** @type {function(module:xxxx, module:xxxx): module:xxxxx} */
function zee() { }
```

Clearly, this has two parameters (not three!), and now we get that right.